Add withEnvFile to support the --env-file option of docker-compose

christopherjwang opened the pull request:
Hey!

I'd like to add support for the `--env-file` docker-compose option. This is useful in environments where a developer has `.env` file that is used for local development (this .env file gets picked up automatically from docker-compose) and the CI/test environment wants to use different values (i.e. ports or environment vars).

fixes #184 

[0] https://github.com/testcontainers/testcontainers-node/issues/184

cristianrgreco commented:
Hi @christopherjwang, looks great, thanks for the contribution! Could you please add an example in the README?

christopherjwang commented:
@cristianrgreco I've updated the PR and added an example. It's now `withEnvFile` (without the s). I am wracking my brain trying to figure out why I thought there could be multiple --env-files (my initial implementation was faulty and incorrect).

christopherjwang commented:
Thanks for the quick turnaround! What's the protocol and timeline for cutting a new version/release?

cristianrgreco commented:
@christopherjwang released in [v8.2.0](https://github.com/testcontainers/testcontainers-node/releases/tag/v8.2.0) 🙂
